app/dashboard/page.module.scss

```diff
@@ -19,6 +19,10 @@
 		display: flex;
 		justify-content: center;
 		margin-top: 20px;
+
+		a {
+			text-decoration: none;
+		}
```

Comment:
najprawdopodobniej to mogłoby być w jakimś common styles, bo najpewniej nie chcesz mieć text-decoration dla każdego linku :)